site/content/en/docs/overview/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Overview"
+linkTitle: "Overview"
+weight: 1
+description: >
+  Why Kueue?
+---
+
+Kueue is a set of APIs and controller for job queueing. It is a job-level manager that decides when a job should be admitted to start (as in pods can be created) and when it should stop (as in active pods should be deleted).
+
+## Why use Kueue
+
+Kueue is a lean controller that you can install on top of a vanilla Kubernetes cluster. Kueue does not replace any existing Kubernetes components. Kueue is compatible with cloud environments where:
+
+* Compute resources are elastic and can be scaled up and down.
+* Compute resources are heterogeneous (in architecture, availability, price, etc.).
+
+Kueue APIs allow you to express:
+
+* Quotas and policies for fair sharing among tenants.
+* Resource fungibility: if a resource flavor is fully utilized, Kueue can admit the job using a different flavor.
+
+The main design principle for Kueue is to avoid duplicating mature functionality in Kubernetes components and well-established third-party controllers. Autoscaling, pod-to-node scheduling and job lifecycle management are the responsibility of cluster-autoscaler, kube-scheduler and kube-controller-manager, respectively. Advanced admission control can be delegated to controllers such as gatekeeper.
```

Comment:
```suggestion
A core design principle for Kueue is to avoid duplicating mature functionality in Kubernetes components and well-established third-party controllers. Autoscaling, pod-to-node scheduling and job lifecycle management are the responsibility of cluster-autoscaler, kube-scheduler and kube-controller-manager, respectively. Advanced admission control can be delegated to controllers such as gatekeeper.
```